osgp/platform/osgp-adapter-domain-publiclighting/src/main/java/org/opensmartgridplatform/adapter/domain/publiclighting/application/config/SchedulingConfigForEventRetrievalScheduledTask.java

```diff
@@ -37,6 +37,7 @@
     private static final String PROPERTY_NAME_SCHEDULING_TASK_EVENT_RETRIEVAL_MANUFACTURER_NAME = "scheduling.task.event.retrieval.manufacturer.name";
     private static final String PROPERTY_NAME_SCHEDULING_TASK_EVENT_RETRIEVAL_MAX_ALLOWED_AGE = "scheduling.task.event.retrieval.max.allowed.age";
 
+    private static final String PROPERTY_NAME_SCHEDULING_TASK_EVENT_RETRIEVAL_BASE_NUMBER = "scheduling.task.event.retrieval.base.number";
```

Comment:
Rename BASE_NUMBER to BACKOFF_MULTIPLIER, similar to the terminology used by ActiveMQ.